x-pack/legacy/plugins/transform/public/app/app_dependencies.tsx

```diff
@@ -7,34 +7,27 @@
 import React, { createContext, useContext, ReactNode } from 'react';
 import { HashRouter } from 'react-router-dom';
 
-import chrome from 'ui/chrome';
-import { metadata } from 'ui/metadata';
-
 import { API_BASE_PATH } from '../../common/constants';
 
 import { setDependencyCache } from '../shared_imports';
 import { AppDependencies } from '../shim';
 
 import { AuthorizationProvider } from './lib/authorization';
 
-const legacyBasePath = {
-  prepend: chrome.addBasePath,
-  get: chrome.getBasePath,
-  remove: () => {},
-};
-const legacyDocLinks = {
-  ELASTIC_WEBSITE_URL: 'https://www.elastic.co/',
-  DOC_LINK_VERSION: metadata.branch,
-};
-
 let DependenciesContext: React.Context<AppDependencies>;
 
 const setAppDependencies = (deps: AppDependencies) => {
+  const legacyBasePath = {
```

Comment:
Running Kibana with a base path, I am seeing this error:

![image](https://user-images.githubusercontent.com/7405507/75358012-2532eb00-58aa-11ea-94de-69589210832c.png)

and I see a 404 in the console which shows a doubling up of the base path `kch`
`GET http://localhost:5601/kch/kch/api/transform/privileges 404 (Not Found)`

Reply:
Fixed in aa8535b.